qdrant-landing/content/articles/qdrant-1.3.x.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Introducing Qdrant 1.3.0"
+short_description: "Check out what Qdrant 1.3 brings to vector search"
+description: "Check out what Qdrant 1.3 brings to vector search"
+social_preview_image: /articles_data/qdrant-1.3.x/social_preview.png
+small_preview_image: /articles_data/qdrant-1.3.x/icon.svg
+preview_dir: /articles_data/qdrant-1.3.x/preview
+weight: 1
+author: David Sertic
+author_link: 
+date: 
+draft: false
+keywords:
+  - vector search
+  - new features
+  - oversampling
+  - grouping lookup
+  - io_uring
+  - appendable mmap
+  - group requests
+---
+
+A brand-new [Qdrant 1.3.0 release](https://github.com/qdrant/qdrant/releases/tag/v1.3.0) comes packed with a plethora of new features, performance improvements and bux fixes:
+
+1. [Oversampling for Quantization:](#oversampling-for-quantization) Improve the accuracy and performance of your queries while using Scalar or Product Quantization
+2. [Asynchronous I/O interface:](#asychronous-io-interface) Reduce overhead by managing I/O operations asynchronously, and by minimizing the need for context switches.
```

Comment:
Would you mind putting the async interface in the first place? To my mind, that is the most significant change.